ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -271,6 +285,14 @@ string id
         }
         progress.Report.Merge(converter.Report);
 
+        // track the object type counts as an event before we try to send
+        // this will tell us the composition of a commit the user is trying to send, even if it's not successfully sent
+        Analytics.TrackEvent(
+          AccountManager.GetDefaultAccount(),
```

Comment:
Using the default account could skew our reporting, as the person doing the operation could be someone else.